go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/docker/buildx
 
-go 1.23.0
+go 1.24.0
```

Comment:
. @thaJeztah wants this to be left to 1.23 I believe

Reply:
Ah yes forgot about it